docs/source/theory_biax.rst

```diff
@@ -0,0 +1,52 @@
+===============================================
+PlanarBiaxialExtension and LayeredPlanarBiaxial
+===============================================
+
+.. figure:: biax.svg
+   :alt:
+   :width: 50.0%
+   :align: center
+
+We consider a planar tissue sample in the :math:`xy` plane (i.e., the
+third coordinate direction is along the small tissue thickness). The
+sample has reference dimensions :math:`L_{10}`, :math:`L_{20}`, and
+:math:`t` along the coordinates 1, 2, and 3, respectively. Furthermore,
+we assume that the material is incompressible. Thus, the deformation
+gradient is:
+
+.. math:: \mathbf{F} = \mathop{\mathrm{diag}}\left[\lambda_1,\lambda_2,\frac{1}{\lambda_1\lambda_2}\right].
+
+The first two directions are in the tissue’s plane, and the third
+direction stress will be zero. Hence, we can calculate :math:`p` by
+equating :math:`\sigma_{33}=0`.
+
+If :math:`\boldsymbol{M}=\left[\cos(\theta),\sin(\theta),0 \right]` lies
+in :math:`1-2` plane (i.e., planar), then the third term does not
```

Comment:
In LaTeX, I'd write 1-2 plane as either $1\text{-}2$ plane or 1-2 plane (since the numbers are numbers and the dash is in text mode). I'd be tempted to try `:math:'1'-:math:'2'` plane or just 1-2 plane.

Reply:
Changed